### DISCUSSION TITLE
Measure the impact of "Perturbation"

### DISCUSSION THREAD

**Discussion by Juneil-Jang** [aristoteleo/dynamo-release#494] [General]
Hi, I appreciate the good tool.
I want to quantify the impact of perturbation of the _in silico_ perturbed data. 
This package have that kind of analysis tool? 
if not, I wonder where I can find the estimated expression level by _in silico_ perturbation. 
Thank you!

**Comment by Xiaojieqiu** [comment]
I am busy with several commitments this week and don't have the time to give you a detailed answer. However, you should read the method section regarding the in silico perturbation in our publication (https://www.sciencedirect.com/science/article/pii/S0092867421015774)  and the following pages in our readthedocs to have a deep understanding of our approach: 

the theory: https://dynamo-release.readthedocs.io/en/latest/notebooks/perturbation_introduction_theory.html
the documentation of the perturbation function: https://dynamo-release.readthedocs.io/en/latest/_autosummary/dynamo.pd.perturbation.html#dynamo.pd.perturbation
some examples on using it: https://dynamo-release.readthedocs.io/en/latest/notebooks/perturbation_tutorial/perturbation_tutorial.html

the newly generated `j_delta_x_perturbation` layer after running the perturbation predictions offers the perturbation effects of each gene in each gene. The values in this matrix quantify the impact of perturbation.

**Reply by Xiaojieqiu** [reply]
@Juneil-Jang also convert this as a discussion which is more proper
